diff --git a/cmake/git_version_update.cmake b/cmake/git_version_update.cmake
index 4c339f2a..9a8fd602 100644
--- a/cmake/git_version_update.cmake
+++ b/cmake/git_version_update.cmake
@@ -18,32 +18,7 @@ else()
     )
 endif()
 
-# Get the latest abbreviated commit hash of the working branch
-execute_process(
-        COMMAND git log -1 --format=%h --abbrev=8
-        WORKING_DIRECTORY ${SOURCE_DIR}
-        OUTPUT_VARIABLE GIT_COMMIT_HASH
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# Check if the git index is dirty
-execute_process(
-        COMMAND git diff-index HEAD --
-        WORKING_DIRECTORY ${SOURCE_DIR}
-        OUTPUT_VARIABLE GIT_DIRTY_STRING
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# flatpak-builder breaks any commands that check the git tree
-if(NOT "${GIT_DIRTY_STRING}" STREQUAL "" AND NOT FLATPAK_BUILD AND NOT FDROID_BUILD)
-    # on CI, display the git diff in the log
-    if(OVERRIDE_GIT_BRANCH OR DEFINED ENV{CI})
-        message("=== git index dirty, see diff:")
-        message("${GIT_DIRTY_STRING}")
-    endif()
-
-    set(GIT_COMMIT_HASH "${GIT_COMMIT_HASH}-dirty")
-endif()
+set(GIT_COMMIT_HASH 5c77552)
 
 if(NOT GIT_COMMIT_HASH)
     set(THEXTECH_GIT_VERSION "CMakeUnknown")
